### ISSUE TITLE
[v6rc5] initialValues are not propagated 

### ISSUE THREAD

**Issue by yury-dymov** [erikras/redux-form#1570]
```
@connect(mapStateToProps)
@reduxForm(...)
export default class extends Component {
  render() {
    console.log(this.props.initialValues) <-- undefined
    ...
  }
}

function mapStateToProps(state){
  return { initialValues: { ...} }
}
```
However in v6rc4 worked fine

**Comment by hathawsh** [comment]
I also noticed that the initialValues prop disappeared recently. I thought the omission was intentional because it doesn't appear to be documented:

http://redux-form.com/6.0.0-rc.4/docs/api/Props.md/

On further reflection, however, I think the initialValues prop really ought to be there.

**Comment by martinratinaud** [comment]
I do have the same problem and rolled back to v6rc4 which works

**Comment by erikras** [comment]
This was fixed by #1598. I will post here again when it is released.

**Comment by erikras** [comment]
Published in [`v6.0.1`](https://github.com/erikras/redux-form/releases/tag/v6.0.1).
